DestinyT.txt

```diff
@@ -1,11 +1,7 @@
 Hello, my name is Destiny Tibbitts.
-Three words that describe me would be caring, motivated, and humble.
-I would say my hoobies are learning to code and being a mom.
-I love being a mom because there is nothing better than seeing my daughter run around outside or try to mimic words or actions.
-Seeing her grow and knowing she looks up to me fills my heart with love.
-Coding is another activity I love because, despite how stressful it can be, and the fact that it's already made me cry, it is very intriguing and evergrowing.
-There are not a lack of jobs for coding, either, which is a big positive as well.
-My favorite animal is an octopus because of how intelligent they are.
-Coding makes me feel like a spy.
-Or like I'm saving the world.
-Both are very nice.
\ No newline at end of file
+Three words that don't describe me would be rich, tall, and lazy.
+I would say my hoobies aren't sports and traveling.
+I don't play sports because I'm to busy being a mom and I'm not athletic at all.
+I couldn't hit the broad side of a barn with... well anything. My aim sucks.
+I don't have a car for traveling.
+I also don't travel because I don't have money for it.
```

Comment:
When you get that awesome developer job, you can take your daughter on all kinds of wonderful trips that she will remember for the rest of her life!